### PR TITLE
Add projects built with JanusGraph [skip ci]

### LINKED ISSUE #1149: Add a listing of software that is built on or depends on JanusGraph
For starters, here are some that I'm aware of:

- [ ] [Apache Atlas](https://github.com/apache/atlas) — metadata management for governance ([website](http://atlas.apache.org/))
- [ ] [Eclipse Keti](https://github.com/eclipse/keti) — access control service to protect RESTful APIs ([website](https://projects.eclipse.org/projects/iot.keti))
- [ ] [Exakat](https://github.com/exakat/exakat) — PHP static analysis ([website](https://www.exakat.io/))
- [ ] [Unifi Catalog & Discovery](https://unifisoftware.com/product/catalog-discovery/)

> ### View Data Relationships
>
> Graphically represented by JanusGraph embedded into the Unifi Data Catalog UI, a user can easily determine how datasets and even attributes are related. Understanding where data comes from, the provenance, and lineage are essential to determining data validity.

- [ ] [Uber Knowledge Graph](https://www.youtube.com/watch?v=C01Gh0g01JE) ([event info](https://leap.ai/events/2017/08/06/uber_knowledge_graph))
- [ ] [Express-Cassandra](https://github.com/masumsoft/express-cassandra) - Cassandra ORM/ODM/OGM for Node.js with optional support for Elassandra & JanusGraph
- [ ] [Windup](https://github.com/windup/windup) by Red Hat — application migration and assessment tool ([website](https://developers.redhat.com/products/rhamt/overview/))

### PR DESCRIPTION
**Pull request by mbrukman**
Fixes #1149; though this is just a starter list and will be continuously
updated.

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there an issue associated with this PR? Is it referenced in the commit message?
- [x] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [/] Have you written and/or updated unit tests to verify your changes?
- [/] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [/] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [/] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [x] Have you ensured that format looks appropriate for the output in which it is rendered?
- [x] If this PR is a documentation-only change, have you added a `[skip ci]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,13 @@ tools:
 Please see [`CONTRIBUTING.md`](CONTRIBUTING.md) for more information, including
 CLAs and best practices for working with GitHub.
 
+## Built with JanusGraph
```
Comment (hsaputra) [inline README.md]: How about rephrase it to `Powered by JanusGraph` ?

Sounded more as stronger statement? ;)

File: README.md [hunk 2]
```diff
@@ -84,6 +84,13 @@ tools:
...
+
+* [Apache Atlas](https://github.com/apache/atlas) - metadata management for governance ([website](http://atlas.apache.org/))
+* [Eclipse Keti](https://github.com/eclipse/keti) - access control service to protect RESTful APIs ([website](https://projects.eclipse.org/projects/iot.keti))
+* [Exakat](https://github.com/exakat/exakat) - PHP static analysis ([website](https://www.exakat.io/))
+* [Windup](https://github.com/windup/windup) by Red Hat - application migration and assessment tool ([website](https://developers.redhat.com/products/rhamt/overview/))
+
```
Comment (pluradj) [inline README.md]: [Open Network Automation Platform (ONAP)](https://www.onap.org/), Linux Foundation project - automation and orchestration for Software-Defined Networks ([website](https://www.onap.org/home/community))

https://gerrit.onap.org/r/gitweb?p=aai/aai-common.git;a=blob;f=aai-core/pom.xml;h=f8761ba6448ab9ebfaa605ad8a254e9498e1de92;hb=refs/heads/beijing#l424

File: README.md (hunk 1 above)
Comment (mbrukman) [reply README.md]: Updated; both seem fine to me.

File: README.md (hunk 2 above)
Comment (mbrukman) [reply README.md]: Thanks! I've been looking at ONAP for a while since I saw them mention JanusGraph, but forgot to add them to this list.

I've added ONAP to my PR; I did not highlight "The Linux Foundation" as we are not highlighting the source of the other projects, and I didn't add the website link since the name already links to the same site.

**Comment by mbrukman** [comment]
@hsaputra  and @pluradj — thanks for your reviews! Note that I added the other projects listed on #1149 to the PR to add them all in one step rather than a few at-a-time.
